modules/ingestor/src/service/advisory/osv/loader.rs

```diff
@@ -459,7 +467,7 @@ async fn create_package_status<C: ConnectionTrait>(
             .await?;
     }
 
-    if let (_, Some(fixed)) = &parsed_range {
+    if let (_, Some((fixed, false))) = &parsed_range {
```

Comment:
Are we ingesting both "affected" and "fixed" statuses every time we find high version? This is not a topic for this PR, but I think we should rethink this in future as it seems like redundant information.